pages/autosuggest/hide-entered-text-label.page.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useRef, useState } from 'react';
+
+import Autosuggest, { AutosuggestProps } from '~components/autosuggest';
+
+const empty = <span>Nothing found</span>;
+const options = [
+  { value: 'Option 0', tags: ['tag1', 'tag2'], filteringTags: ['bla', 'opt'], description: 'description1' },
+  { value: 'Option 1', labelTag: 'This is a label tag' },
+  { value: 'Option 2' },
+  { value: 'Option', description: 'description2' },
+];
+const enteredTextLabel = (value: string) => `Use: ${value}`;
+export default function AutosuggestPage() {
+  const [value, setValue] = useState('');
+  const [hasOptions, setHasOptions] = useState(true);
+  const [hasEmpty, setHasEmpty] = useState(true);
+  const [hideEnteredTextLabel, setHideEnteredTextLabel] = useState(false);
+  const ref = useRef<AutosuggestProps.Ref>(null);
+  return (
+    <div style={{ padding: 10 }}>
+      <h1>Hide entered text labebel in autosuggest</h1>
+      <Autosuggest
+        ref={ref}
+        value={value}
+        readOnly={false}
+        options={hasOptions ? options : []}
+        onChange={event => setValue(event.detail.value)}
+        enteredTextLabel={enteredTextLabel}
+        hideEnteredTextLabel={hideEnteredTextLabel}
+        ariaLabel={'hide entered text label in autosuggest'}
+        selectedAriaLabel="Selected"
+        empty={hasEmpty ? empty : null}
+      />
+      <button id="hide-entered-text-label" onClick={() => setHideEnteredTextLabel(true)}>
```

Comment:
nit: these can be toggle buttons, you can use the Cloudscape Toggle component for this. This way one can see the current state and change it w/o need to reload page

Reply:
This will be much easier to understand, thanks! I'll update this PR to add it.